scripts/ci/release_authority_snapshot.py

```
@@ -0,0 +1,172 @@
+
+import argparse
+import json
+from pathlib import Path
+import subprocess
+import sys
+from typing import Any
+
+
+def _gh_json(repository: str, endpoint: str) -> dict[str, Any] | list[Any]:
+    """Read one GitHub REST endpoint without exposing subprocess diagnostics."""
+    completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)